.gitignore

```diff
@@ -1,4 +1,5 @@
 *~
+*pyc
 __pycache__
```

Comment:
I added it because there was a .pyc file lying around in my working copy,but I'm using python3